whitepaper/chapters/unconfirmed.tex

```diff
@@ -15,8 +15,8 @@ \section{Unconfirmed Transactions}
 \begin{enumerate}
 	\item{A client sends a new transaction directly to the node.}
```

Comment:
i am not sure if these should lead with articles "A"

Reply:
I would keep them, makes the sentence complete and easier to read.